ci/.dockerignore

```diff
@@ -5,9 +5,11 @@
 # Note: This file does not prevent us from accessing other files from within a
 # running docker *container*. Those will be availble from our mounted volume.
 # https://docs.docker.com/engine/reference/builder/#dockerignore-file
+
+# Start by ignoring everything, then only allow a few things.
```

Comment:
TIL: nice.

Reply:
That `*` was already there. I just documented what was happening for myself and others. I assumed you put it there, so maybe you forgot that you already new it? :-)